xcm/xcm-builder/src/origin_aliases.rs

```diff
@@ -15,65 +15,32 @@
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
 use frame_support::traits::{ContainsPair, Get};
+use sp_std::marker::PhantomData;
 use xcm::latest::prelude::*;
 
-pub struct AliasSiblingAccountId32<ParachainId>(sp_std::marker::PhantomData<ParachainId>);
-impl<ParachainId: Get<u32>> ContainsPair<MultiLocation, MultiLocation>
-	for AliasSiblingAccountId32<ParachainId>
+pub struct RemovePrefixAccountId32<Prefix>(PhantomData<Prefix>);
```

Comment:
I'd probably name it `AliasForeignAccountId32` instead of mentioning anything about removing prefixes.